components/utilities/headerLink.module.css

```diff
@@ -10,6 +10,10 @@
   @apply font-bold;
 }
 
+.HeaderContainer code {
```

Comment:
Can we add classes in the HTML for the `<code>` tag? Or additional `<span>` elements inside the `<h1>`? We should find a way to only add the `px-1` and `mx-1` classes to `<code>` tags that are in the middle of the heading, like so:

```
<!-- These should have px-1 and mx-1 -->
<h1>I'm a nice `<code>` heading</h1>
```

And not to `<code>` tags that show up right after the `<h1>` tag, like so:

```
<!-- These code tags should only have margin at the right, since margin left will throw the alignment off -->
<h1><code>st.</code>experimental_singleton</h1>
```

See here:

![Screen Shot 2022-09-13 at 08 05 33](https://user-images.githubusercontent.com/34423371/189889813-4986a1ab-b28d-4ba3-b3ee-e9b617cd6201.png)

Reply:
Really good points! I hadn't considered the consequences of blindly applying `px-1` and `mx-1` to headings. 

> Can we add classes in the HTML for the \<code> tag?

Where should this be done? In this same `.css` file or elsewhere?

> And not to \<code> tags that show up right after the \<h1> tag, like so

Same noob question as above. 😅  Can conditional application of `px-1` and `mx-1` based on the position of \<code> tags be done in CSS?

Reply:
I was referring to turning the `.md` heading tag with the inline tag:

```
## Set up your `virtual` environment
```

into something like this on the HTML:

```
<h2><span>Set up your</span><code>virtual</code><span>environment</span></h2>
```

To make it easier to style, but that didn't work, so I ended up wrapping the `children` element in `headerLink` in a `<span>` when there's a `<code>` tag present, which helps with the spacing of words without having to rely on `px-1 mx-1`. Here's the result:

![Screen Shot 2022-09-14 at 3 03 39 PM](https://user-images.githubusercontent.com/34423371/190231845-142933b1-2609-4f64-8fd5-0057e4a43d4d.png)

Notice how `Prerequisites` is properly aligned, and `virtual` is spaced out from the rest of the words. Hope this helps!

Reply:
> into something like this on the HTML
Ah, I didn't occur to me to try that! 

Your latest changes by wrapping the `children` element in `headerLink` in a `<span>` when there's a `<code>` tag present work well! Thanks for the more robust fix 😃